site/src/sphinx/server-access-log.rst

```diff
@@ -163,6 +174,119 @@ the condition.
+|                               | until the response processing ended, in milliseconds               |
++-------------------------------+--------------------------------------------------------------------+
+| ``totalDurationNanos``        | the amount of time taken since the request processing started and  |
+|                               | until the response processing ended, in nanoseconds                |
++-------------------------------+--------------------------------------------------------------------+
+| ``sessionProtocol``           | the session protocol of the request.                               |
+|                               | e.g. ``h1``, ``h2``, ``h1c`` or ``h2c``                            |
++-------------------------------+--------------------------------------------------------------------+
+| ``serializationFormat``       | the serialization format of the request.                           |
+|                               | e.g. ``tbinary``, ``ttext``, ``tcompact``, ``tjson`` or ``none``   |
++-------------------------------+--------------------------------------------------------------------+
+| ``scheme``                    | the scheme value printed as ``serializationFormat+sessionProtocol``|
++-------------------------------+--------------------------------------------------------------------+
+| ``host``                      | the host name of the request                                       |
++-------------------------------+--------------------------------------------------------------------+
+| ``status``                    | the status code and its reason phrase of the response.             |
+|                               | e.g. ``200 Ok``                                                    |
++-------------------------------+--------------------------------------------------------------------+
+| ``statusCode``                | the status code of the response. e.g. ``200``                      |
++-------------------------------+--------------------------------------------------------------------+
+
+.. _timestamp-format:
+
+Customizing timestamp format
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can specify a new date/time format for the ``%t`` token with DateTimeFormatter_. You can use one of the
+following formatter which is provided by JDK as a variable of the ``%t`` token, e.g. ``%{BASIC_ISO_DATE}t``.
+If you want to use your own pattern, you can specify it as the variable, e.g. ``%{yyyy MM dd}t``.
+
++-------------------------+----------------------------------+--------------------------------------------+
+| Formatter               | Description                      | Example                                    |
++=========================+==================================+============================================+
+| ``BASIC_ISO_DATE``      | Basic ISO date                   | ``20111203``                               |
```

Comment:
Date?

Reply:
Actually I copied this table from javadoc: https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html
I don't know why they capitalize some words.. 

Does it better to change the description? WDYT?

Reply:
Then please forget my comments :)